### PR TITLE
chore: remove grainfind

### PR DESCRIPTION
**Pull request by ospencer**

### CODE REVIEW

File: test-suite/unit-tests/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "description": "",
   "scripts": {
-    "build": "glob-exec --foreach \"contracts/*.gr\" -- \"node script/grainfind.js compile {{file}}\"",
+    "build": "glob-exec --foreach \"contracts/*.gr\" -- \"grain compile -I ../../ --wasi-polyfill ../../wasi.gr --no-bulk-memory --use-start-section --no-gc --elide-type-info --release {{file}}\"",
```
Comment (phated) [inline test-suite/unit-tests/package.json]: Do we also want to add a .vscode/settings.json file that sets these flags in the LSP?

Comment (ospencer) [reply test-suite/unit-tests/package.json]: done

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest]
+        node-version: ["16"]
+        grain-version: ["v0.5.11"]
+
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v2
```
Comment (phated) [inline .github/workflows/ci.yml]: We want v3 here because of the GHA deprecations (I believe).

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3.1.1
```
Comment (phated) [inline .github/workflows/ci.yml]: Can probably use a newer version of this unless you were trying to align things (?).. Might want to use the versions referenced by binaryen.ml or something?

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+      - name: Install Grain
+        uses: engineerd/configurator@v0.0.8
+        with:
+          name: grain
+          url: https://github.com/grain-lang/grain/releases/download/grain-${{ matrix.grain-version }}/grain-linux-x64
+
+      - name: Install dependencies
+        run: |
+          npm ci && cd test-suite && npm ci && cd example-contracts && npm ci
```
Comment (phated) [inline .github/workflows/ci.yml:40]: lol we really need to fix the workspaces, as the top-level `npm ci` should be installing every dep.

File: test-suite/unit-tests/tests/economics.test.js [hunk 5]
```diff
@@ -39,5 +39,5 @@ test("prepaidGas", async t => {
 test("usedGas", async t => {
   const {root, contract} = t.context.accounts
   const res = await root.call(contract, "doUsedGas", {})
-  t.deepEqual(res, 274214078913)
+  t.deepEqual(res, 274446183237);
```
Comment (phated) [inline test-suite/unit-tests/tests/economics.test.js]: Is this flakey? I thought philip changed this number in his final commit?

**Review by phated — APPROVED** [review]
Saw a couple things. Up to you which ones you think are worthwhile @ospencer

File: test-suite/unit-tests/tests/economics.test.js (hunk 5 above)
Comment (ospencer) [reply test-suite/unit-tests/tests/economics.test.js]: This is more or less going to change with every release of Grain. We should really be testing for a reasonable range.

**Pull request merged**